Source/Engine/Platform/Base/PlatformBase.h

```diff
@@ -316,6 +316,25 @@ DECLARE_SCRIPTING_TYPE_MINIMAL(PlatformBase);
     /// <param name="ptr">A pointer to the memory block to deallocate.</param>
     static void Free(void* ptr) = delete;
 
+    /// <summary>
+    /// Returns the OS's default page size that can be used with AllocatePages.
+    /// </summary>
+    static uint64 GetDefaultPageSize() = delete;
```

Comment:
There is already API to get page size on system: `Platform::GetCPUInfo().PageSize` so we can skip that one.

Reply:
Yes, missed that one. Fixed.